packages/internal/src/config.js

```diff
@@ -31,9 +31,10 @@ export const BridgeId = /** @type {const} */ ({
 });
 harden(BridgeId);
 
+/** @satisfies {Record<string, BridgeId>} */
 export const CosmosInitKeyToBridgeId = {
```

Comment:
harden this while we're here?

slippery slope, I suppose. This PR is already pretty big. And I think so far I haven't see any changes that have runtime impact.